pom.xml

```diff
@@ -725,6 +726,12 @@
       <version>${swagger.ui.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.javassist</groupId>
+      <artifactId>javassist</artifactId>
+      <version>3.27.0-GA</version>
+    </dependency>
```

Comment:
See https://github.com/dropwizard/dropwizard/issues/3084#issuecomment-569142984 for why this is necessary